.github/workflows/docker.yml

```diff
@@ -0,0 +1,58 @@
+name: Docker CI/CD
+
+on:
+  schedule:
```

Comment:
It was something that we have done with upstream images to ensure that we get to know if something breaks (like on FLINT and FLINT.Example). Hence we add this additional step to build and push the images on a cron schedule. Let me know if we can remove this in case of Taswira.

Reply:
Got it. While that sounds like a good step, I don't think it's required for taswira because there's nothing in it that could break overnight. I think, things will only break after a change. Do you agree?